core/examples/abuse/README.md

```diff
@@ -1,177 +0,0 @@
-# Backpressure and peer abuse example
```

Comment:
Same question, why is it deleted? Seems to not be tied to the changes in this PR.

Reply:
I'm OK with removing this. It's quite rotten.